scalardb-cluster-standalone-mode/scalardb-cluster-node.properties

```diff
@@ -1,7 +1,46 @@
-scalar.db.storage=jdbc
-scalar.db.contact_points=jdbc:postgresql://postgresql:5432/postgres
-scalar.db.username=postgres
-scalar.db.password=postgres
+# For MySQL
```

Comment:
Added configurations for MySQL, Oracle Database, SQL Server, DynamoDB Local, and Cassandra.